manager/src/main/scala/org/broadinstitute/transporter/transfer/TransferController.scala

```diff
@@ -60,9 +60,28 @@ class TransferController(
     offset: Long,
     limit: Long,
     newestFirst: Boolean
-  ): IO[List[RequestSummary]] =
-    lookupSummaries(Left((offset, limit)), newestFirst)
-      .transact(dbClient)
+  ): IO[List[RequestSummary]] = {
+    val query = for {
+      ids <- List(
+        Fragment.const(s"select id from $RequestsTable order by received_at"),
```

Comment:
Nitpick: If possible capitalize the keywords for readability and syntax.

Reply:
+1 for a separate PR that fixes this in all of Transporter

Note to self: make a ticket to do ^^^

Reply:
https://broadinstitute.atlassian.net/browse/DSPDC-521

Reply:
https://broadinstitute.atlassian.net/browse/DSPDC-521